src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nit: let's call this `StreamTests.cs` to handle roundtrip stream cases since we already have `Stream.*Tests.cs`.

Reply:
Since we have other code paths to allow "partial progress"/continuation via the Utf8JsonReader overload, something a bit more generic as the filename that covers more uses, might actually be better than using `StreamTests`. 

Something like `ContinuationTests` seems reasonable, but **only** if it makes sense to add test cases for other overloads to this file too. See https://github.com/dotnet/runtime/pull/42158#discussion_r488907272